math.py

```
@@ -0,0 +1,12 @@
+def div(x:int)->int:
+  return x//2
+
+def Modular_division(x:int):
+  return x%2
+
+a=25
+
+print(div(a))
+print(Modular_div(a))
```

Ruff (0.12.2)

Undefined name `Modular_div`

(F821)